src/components/ItemDialogContent.js

```diff
@@ -36,14 +36,23 @@ const iconGithub = <svg viewBox="0 0 24 24">
     14.5,21C14.5,21.27 14.66,21.59 15.17,21.5C19.14,20.16 22,16.42 22,12A10,10 0 0,0 12,2Z" />
     </svg>
 
-const cncfTag = function(cncfRelation) {
+const cncfTag = function(cncfRelation, cncfMember) {
   const text = _.find(fields.cncfRelation.values, {id: cncfRelation}).tag;
   if (cncfRelation === false) {
     return null;
   }
   if (cncfRelation === 'member') {
+    const label = {
+      platinum: 'CNCF Platinum Member',
+      gold: 'CNCF Gold Member',
+      silver: 'CNCF Silver Member',
+      academic: 'CNCF Academic Member',
+      nonprofit: 'CNCF Non Profit Member',
+      linux_foundation: 'LF Project',
+      cncf: 'CNCF Project'
+    }[cncfMember] || 'CNCF Member';
```

Comment:
Could we do a fatal error here rather than saying `CNCF Member`? There's no other scenario where you should be hitting this.

Reply:
Actually, this can not even happen